utils/src/wizard/NoExecuteStep.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from '../..';
+import { UserCancelledError } from '../errors';
+import { AzureWizardExecuteStep } from './AzureWizardExecuteStep';
+
+export class NoExecuteStep<T extends IActionContext> extends AzureWizardExecuteStep<T> {
+    public priority: number = 200;
+
+    private _key: string = "NoExecute";
+    public async execute(_wizardContext: T): Promise<void> {
+        throw new UserCancelledError(this._key);
```

Comment:
Should we create a new Error class for it?

Reply:
I don't think for today that will be helpful, or even if the answer is yes. There's a lot of existing code which basically says "ok, no big deal, just a user cancelled error", and that's the behavior we want. If we want additional behavior, then ya, I think an answer of "yes" becomes more clear.

Reply:
It might be nice to have an additional error just for telemetry purposes.